communitybsl.properties

```diff
@@ -1,15 +1,23 @@
 category=Languages
 description=Code Analyzer for 1C (BSL)
 homepageUrl=https://1c-syntax.github.io/sonar-bsl-plugin-community/en
-archivedVersions=1.11.0,1.12.0,1.13.0,1.14.0,1.15.0,1.15.1
-publicVersions=1.15.2
+archivedVersions=1.11.0,1.12.0,1.13.0,1.14.0,1.15.0,1.15.1,1.15.2
+publicVersions=1.16.1
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-communitybsl-plugin
 
+1.16.1.description=Performance optimization, new diagnostics
+1.16.1.sqs=[10.8,LATEST]
+1.16.1.sqcb=[24.12,LATEST]
+1.16.1.sqVersions=[9.9,10.7]
+1.16.1.date=2025-03-25
+1.16.1.changelogUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/tag/v1.16.1
+1.16.1.downloadUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/download/v1.16.1/sonar-communitybsl-plugin-1.16.1.jar
+
 1.15.2.description=Support SQ 25.1+
-1.15.2.sqs=[10.8,LATEST]
-1.15.2.sqcb=[24.12,LATEST]
+1.15.2.sqs=[10.8,2025.1]
```

Comment:
End this range with `2025.1.*` to pick up all future point releases (2025.1.1 is already out, BTW)

Reply:
fixed